sqlalchemy_searchable/__init__.py

```diff
@@ -347,7 +347,8 @@ def sync_trigger(
     tsvector_column,
     indexed_columns,
     metadata=None,
-    options=None
+    options=None,
+    update_rows=True,
```

Comment:
I set the default value to `True` for backwards compatibility, but I think the default should really be `False`, since an out-of-date index is better than a locked table.